damusTests/MarkdownTests.swift

```diff
@@ -21,47 +21,35 @@ class MarkdownTests: XCTestCase {
     }
 
     func test_convert_link() throws {
-        let helper = Markdown()
-        let md = helper.process("prologue https://nostr.build epilogue")
+        let md = Markdown.parse(content: "prologue https://nostr.build epilogue")
         let expected = try AttributedString(markdown: "prologue [https://nostr.build](https://nostr.build) epilogue", options: md_opts)
         XCTAssertEqual(md, expected)
     }
-
-    func test_convert_link_no_scheme() throws {
-        let helper = Markdown()
-        let md = helper.process("prologue damus.io epilogue")
-        let expected = try AttributedString(markdown: "prologue [damus.io](https://damus.io) epilogue", options: md_opts)
+    
+    func test_no_convert_markdown_link() throws {
+        let md = Markdown.parse(content: "prologue [link](https://nostr.build) epilogue")
+        let expected = try AttributedString(markdown: "prologue [link](https://nostr.build) epilogue", options: md_opts)
         XCTAssertEqual(md, expected)
     }
-
-    func test_convert_links() throws {
-        let helper = Markdown()
-        let md = helper.process("prologue damus.io https://nostr.build epilogue")
-        let expected = try AttributedString(markdown: "prologue [damus.io](https://damus.io) [https://nostr.build](https://nostr.build) epilogue", options: md_opts)
+    
+    func test_no_convert_with_emoji() throws {
+        let md = Markdown.parse(content: "test link w/ emoji ❤️ in a [string](https://nostr.build)")
```

Comment:
Did you mean to put the heart within the `[…]`?

Reply:
Yeah, emojis can mess up the character counts so I think it's important to test. They don't always fit into one utf16 character so the counts can be off.

Reply:
got it! We should make sure we have a character with unicode surrogates! Got this one from Wikipedia: [𐐷](https://en.wikipedia.org/wiki/%F0%90%90%8F)

Reply:
yeah, good call